Makefile

```diff
@@ -1605,42 +1605,41 @@ llama-q8dot: pocs/vdot/q8dot.cpp ggml/src/ggml.o \
 # Mark legacy binary targets as .PHONY so that they are always checked.
 .PHONY: main quantize perplexity embedding server
 
+# Define the object file target
+examples/deprecation-warning/deprecation-warning.o: examples/deprecation-warning/deprecation-warning.cpp
+	$(CXX) $(CXXFLAGS) -c $< -o $@ $(LDFLAGS)
```

Comment:
Ugh I missed one. I think LDFLAGS is not needed here, it's only required when creating libraries or binaries, for object files CXXFLAGS is enough.

Reply:
Very helpful, thank you!!